packages/teuchos/parameterlist/src/Teuchos_ParameterList.hpp

```diff
@@ -965,11 +965,11 @@ ParameterList& ParameterList::set(
 template<typename T, typename S, typename>
 inline
 ParameterList& ParameterList::set(
-  std::string const& name_in, S&& value_in, std::string const& docString_in,
+  std::string const& name_in, const S& value_in, std::string const& docString_in,
```

Comment:
Same here.  Now that this is a `const S&`, this will not enable move semantics to allow more efficient moving of data into a PL.